package.json

```diff
@@ -25,7 +25,11 @@
     "./generator": {
       "require": "./dist-cjs/entrypoints/generator.js",
       "import": "./dist-esm/entrypoints/generator.js"
-    }
+    },
+    "./_/*": {
+      "require": "./dist-cjs/*.js",
+      "import": "./dist-esm/*.js"
```

Comment:
This allows access to the implementation which is needed when using a custom output directory.